xl2times/transforms.py

```diff
@@ -6,17 +6,30 @@
 from functools import reduce
 from itertools import groupby
 from pathlib import Path
+from typing import Callable
 from typing import Dict, List, Set
 
 import pandas as pd
 from loguru import logger
 from more_itertools import locate, one
 from pandas.core.frame import DataFrame
+from tqdm import tqdm
 
 from . import datatypes
 from . import utils
 from .utils import max_workers
 
+query_columns = {
+    "pset_set",
+    "pset_pn",
+    "pset_pd",
+    "pset_ci",
+    "pset_co",
+    "cset_set",
+    "cset_cn",
+    "cset_cd",
+}
+
```

Comment:
Do we need them back? We made them table specific in #172

Reply:
Ah sorry, must have stuffed up the merge.  will remove again.